File: 01.ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P10_MultiplicationTable2.java
import java.util.Scanner;
public class P10_MultiplicationTable2 {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        int number = Integer.parseInt(console.nextLine());
        int multiplier = Integer.parseInt(console.nextLine());

        do {
            System.out.format("%d X %d = %d" + System.lineSeparator(),
                             number, multiplier, number * multiplier);
            multiplier++;
        } while(multiplier <= 10);
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P10_MultiplicationTable2/P10_MultiplicationTable2.cs
using System;

namespace P10_MultiplicationTable2
{
    class P10_MultiplicationTable2
    {
        static void Main(string[] args)
        {
            int number = int.Parse(Console.ReadLine());
            int multiplier = int.Parse(Console.ReadLine());

            do
            {
                Console.WriteLine($"{number} X {multiplier} = {number * multiplier}");
                multiplier++;
            } while (multiplier <= 10);
        }
    }
}
